lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -499,4 +518,24 @@ private function getCreateTableSQL(string $table) : ?string
             [$table]
         ) ?: null;
     }
+
+    /**
+     * @param string $tableName
+     *
+     * @return Table
+     */
+    public function listTableDetails($tableName)
+    {
+        $table = parent::listTableDetails($tableName);
+
+        $tableCreateSql = $this->getCreateTableSQL($tableName) ?? '';
```

Comment:
Is it possible that the method will return NULL? It should always return an array.

Reply:
The signature of the `getCreateTableSQL` method says otherwise: https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php#L483 :disappointed:

Reply:
Please use permalinks instead of `master`. The link is already outdated.

Reply:
Edit previous comment with permalink. Sorry for that. I just learned about the 'y' keyboard shortcut to get those permalinks. :)